enterprise/server/util/proxy_util/BUILD

```diff
@@ -0,0 +1,11 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+go_library(
+    name = "proxy_util",
+    srcs = ["proxy_util.go"],
+    importpath = "github.com/buildbuddy-io/buildbuddy/enterprise/server/util/proxy_util",
+    visibility = ["//visibility:public"],
+    deps = ["@org_golang_google_grpc//metadata"],
+)
+
+package(default_visibility = ["//enterprise:__subpackages__"])
```

Comment:
I think package visibility declarations are supposed to go at the top of the BUILD file, like this: https://github.com/buildbuddy-io/buildbuddy/blob/dcbf7c4bd743f0a08aebf5623e1edc1b740b2466/enterprise/server/byte_stream_server_proxy/BUILD#L3 -- thought buildfix would fix this, but maybe not?